fill/proto/interactive_filler.go

```diff
@@ -152,6 +152,8 @@ func (r *resolver) resolveField(f *desc.FieldDescriptor) error {
 			)
 			return msgr.resolve()
 		case descriptorpb.FieldDescriptorProto_TYPE_ENUM:
+			prefix := r.makePrefix(f)
+			fmt.Println(r.color.ForegroundANSI(), prefix, "\033[0m")
```

Comment:
What do you think that the prompt displays only `prefix` instead of `message` as the message of `prompt.Select` ?
Other prompts which read primitive (non-enum) input only display `prefix` as its prefix.

Reply:
Not replace, just append. I want to know which field is being selected because I have multiple fields of the same type in a proto message body.

Reply:
Oh, I'm sorry. I mean, I thought it's better to replace the current prefix string (the fully-qualfiied message name) with `prefix` variable's one (the fully-qualified field name).
Since other prompts only display the string of `prefix` variable, I'd like to do it this way for the sake of consistency. Maybe, it's enough to know about the fully-qualified field name, not the fully-qualified message name.